node/router/tests/common/mod.rs

```diff
@@ -110,6 +112,7 @@ pub async fn validator(listening_port: u16, max_peers: u16) -> TestRouter<Curren
         &[],
         max_peers,
         true,
+        false,
```

Comment:
@vicsn should this be `true`? I'm inclined to leave this one as false as this is the default for validators.

Reply:
Indeed, this should remain false, thank you for confirming.